CRM/Contract/Form/EditMembership.php

```diff
@@ -27,6 +27,14 @@ public function preProcess () {
 
     $this->set('membership', $membership);
     $this->set('membership_channels', CRM_Contract_FormUtils::getOptionValueLabels('contact_channel'));
+
+    $contract_file = Api4\File::get(FALSE)
+      ->addWhere('id', '=', $membership['membership_general.contract_file'])
+      ->addSelect('id', 'file_name', 'url')
```

Comment:
`url` also doesn't seem to work in 5.69 (even though the API parameter exists 😑), but I think we can live with that since most users are gonna get the file via "View" anyway, so I'm gonna pretend I didn't see that. 😬